ci-operator/config/openshift/azure-disk-csi-driver-operator/openshift-azure-disk-csi-driver-operator-release-4.10.yaml

```diff
@@ -61,9 +61,10 @@ tests:
     cluster_profile: azure4
     workflow: openshift-e2e-azure-csi-migration
 - as: e2e-azurestack-csi
+  cluster: build01
```

Comment:
Why do we specify cluster name explicitly? Generally its a bad idea as it makes more complicated for Test Platform team to clear jobs from this cluster if it breaks

Reply:
@vrutkovs Thank you for looking into this. Currently only build01 cluster could support vpn connect to the azure stack hub environment, so we have to add it until testplatform team could add support in other build farms as well.
Details ->
https://redhat-internal.slack.com/archives/CBN38N3MW/p1734609899937729?thread_ts=1734590634.106909&cid=CBN38N3MW

Reply:
This needs to be solved with Test Platform ideally - see https://docs.ci.openshift.org/docs/how-tos/adding-a-cluster-profile/#cluster-profile

Reply:
Good to know, double check with Test Platform team, they have a big epic to solve the issue!
https://redhat-internal.slack.com/archives/CBN38N3MW/p1735912917834009?thread_ts=1734590634.106909&cid=CBN38N3MW